plugins/macro/macro.pl

```diff
@@ -153,6 +153,11 @@ sub checkConfig {
 		warning "[macro] orphans: using method 'terminate'\n";
 		configModify('macro_orphans', 'terminate')
 	}
+	
+	if (!defined $::config{macro_defaultPriority} || $::config{macro_defaultPriority} !~ /^\d+$/) {
+		warning "[macro] default priority: using 0 (\"check first\")\n";
+		configModify('macro_defaultPriority', '0');
+	}

```

Comment:
I think for most stuff we just use (and should use) the default value without warnings and modifying `%config` and config file.